frontend/src/components/tests/CaseCreator.test.js

```diff
@@ -4,9 +4,15 @@ import React from "react";
 import { Link } from "react-router-dom";
 import CaseCreator from "../CaseCreator.js";
 
+const mockedUsedNavigate = jest.fn();
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useNavigate: () => mockedUsedNavigate,
+}));
+
 test("renders case creator layer", () => {
-  // render(<CaseCreator />);
-  // const textElement = screen.getByText(/Assurance/i);
-  // expect(textElement).toBeInTheDocument();
+  render(<CaseCreator />);
+  const textElement = screen.getByText(/Create a new assurance case/i);
```

Comment:
What's the significance of the slash notation in the getByText argument vs quotes used in e.g. CaseContainer.test.js?